src/PhappCommandBase.php

```diff
@@ -108,11 +114,16 @@ protected function getPhappEnviromentVariables($directory = './', $manifest = NU
       return $env_vars;
     }
 
+    $data = '';
     // Extract env vars from dotenv files.
     foreach ($finder as $file) {
+      // Extend data so next parsing has all variables declared before.
+      $data .= file_get_contents($file->getPathname());
       // Add dotenv vars.
       $dotenv = new Dotenv();
-      $env_vars = array_replace($env_vars, $dotenv->parse(file_get_contents($file->getPathname()), $file->getPathname()));
+      // Parse dotenv vars.
+      $parsed_env_vars = $dotenv->parse($data, $file->getPathname());
+      $env_vars = array_replace($env_vars, $parsed_env_vars);
```

Comment:
appending the data string and having the array replace is duplicated? maybe just the array_replace() call ordering needs a fix? Else, the appending the data string is fine, but then parsing should be done one time at the end. Adding the same variables multiple times seems weird and is a bit confusing.